<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectValidation</class>
 <widget class="QDialog" name="ProjectValidation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>工程规范检查</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="grpProjectStandards">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="title">
        <string>工程规范</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="cbxWrongCase">
          <property name="text">
           <string>不符合大小写规范</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxWrongNameLength">
          <property name="text">
           <string>名称长短不符合规范</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxColorTypeMismatch">
          <property name="text">
           <string>颜色与对象的类别不一致</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxNoRefNotesOrColorTag">
          <property name="text">
           <string>引用备注/颜色标记缺失或错误</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpDesignStructure">
       <property name="title">
        <string>设计缺陷</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="cbxNestedWorkUnit">
          <property name="text">
           <string>WorkUnit多层嵌套</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoRepeat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxRedundantActorMixer">
          <property name="text">
           <string>ActorMixer无实际调整</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxSingleChildContainer">
          <property name="text">
           <string>Container仅包含一个子节点</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxEmptyNode">
          <property name="text">
           <string>对象不包含有效内容</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxDuplicatedParameter">
          <property name="text">
           <string>RTPC重复绑定内部属性</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpSoundLocalization">
       <property name="title">
        <string>样本与本地化</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="cbxSourceNameDifferent">
          <property name="text">
           <string>源文件与Sound命名不一致</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxSourceNotImported">
          <property name="text">
           <string>未导入或缺失语言</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxSilentPlaceholder">
          <property name="text">
           <string>使用静音临时替代资源</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxSoundNotInEvent">
          <property name="text">
           <string>音效未被事件包含</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxInvalidTrimRange">
          <property name="text">
           <string>音频裁剪超出范围</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpEventBank">
       <property name="title">
        <string>Event与Bank</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="cbxEventDuplicatedByNetRole">
          <property name="text">
           <string>存在1P/3P两个事件</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxWrongEventName">
          <property name="text">
           <string>Event与播放对象命名冲突</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxEmptyEvent">
          <property name="text">
           <string>Event缺少动作或目标</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxEventNotInBank">
          <property name="text">
           <string>Event未被任何Bank包含</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxAbnormalBankSize">
          <property name="text">
           <string>Bank过大或过小</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpMistakes">
       <property name="title">
        <string>粗心大意</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QCheckBox" name="cbxUnassignedToBus">
          <property name="text">
           <string>未指定对应类别的Bus</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxUnassignedSwitchContainer">
          <property name="text">
           <string>SwitchContainer未分配</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxSwitchTo1PByDefault">
          <property name="text">
           <string>SwitchContainer默认播放1P</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxMissingAttenuation">
          <property name="text">
           <string>3D音效未设置衰减曲线</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxSoundNotStreamed">
          <property name="text">
           <string>音乐环境语音未Streaming</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnSelectAll">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>全选</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeselectAll">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>全不选</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnValidateEntireProject">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>检查整个工程（可能会很慢）</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnValidateListEntries">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>检查主窗口列表对象</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnValidateProjectSelection">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>检查Wwise工程选中对象</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSendResultToMainWindow">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>将列表选中项发送到主窗口</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAutoFixSelection">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>一键修复列表选中项</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tblValidationResults">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>类别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>问题</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>对象类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>对象路径（双击可在Wwise工程中查看）</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
